org.mixedrealitytoolkit.core/Utilities/FindObjectUtility.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using System;
+
+namespace MixedReality.Toolkit
+{
+    /// <summary>
+    /// A static utility used to avoid deprecated Find Object functions in favor of replacements introduced in Unity >= 2021.3.18. 
+    /// </summary>
+    public static class FindObjectUtility
+    {
+
+        /// <summary>
```

Comment:
Comment Style.

Can you the unity version information in a `<remarks>` tag.  And in the `<summary>` tag don't call out the implementation details, instead describe what the functions do in more general terms.

Reply:
will do